news/2019-04-08-project-loved-week-of-april-7th.md

```diff
@@ -0,0 +1,225 @@
+
+[![](/wiki/shared/news/2019-04-08-project-loved-week-of-april-7th/osu/ghost.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Camellia - GHOST](https://osu.ppy.sh/beatmapsets/665530#osu) by [fanzhen0019](https://osu.ppy.sh/users/418699), [rustbell](https://osu.ppy.sh/users/227717) and [Callionet](https://osu.ppy.sh/users/3072921)  
+220 BPM, 5:38 | 8.98★  
+*written by [-Roxas](https://osu.ppy.sh/users/1986262)*
+
+Now here's a real killer of a map: fanzhen0019's "Camellia - Ghost" is a real challenge, and by that I mean... are you able to pass this map? With the difficulty hitting nearly 9 stars, it's definitely not one to laugh at. Don't underestimate this, even if it is HP 0, as the map consists of large jumps, fast sliders, spaced high-BPM streams and even some 2B to top it all off. Best of luck to the players that attempt to pass this very scary map.
+
+[![](/wiki/shared/news/2019-04-08-project-loved-week-of-april-7th/osu/haelequin-extended-ver.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[orangentle / Yu\_Asahina - HAELEQUIN (Extended ver.)](https://osu.ppy.sh/beatmapsets/307677#osu) by [DoKito](https://osu.ppy.sh/users/537084)  
+172 BPM, 4:26 | 6.49★, 7.35★, 7.97★  
+*written by [waywern2012](https://osu.ppy.sh/users/5870453)*
+
+The song "HAELEQUIN", coming from BMS, then remixed to SVDX II, has been mapped in osu! many times, and this was one of the earlier maps. The map has grown in popularity because of its three BPM variations on the map. The maps are the same in every difficulty, but the speed changes, making the player more strained on higher difficulties. With the BPM ranging from 172 to 220, this makes the map fairly spread out for the higher skilled among us. With interesting old-style patterns and tricky streams, this map is worth a try!. See what speed you can handle!
+
+[![](/wiki/shared/news/2019-04-08-project-loved-week-of-april-7th/osu/usatei.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[IOSYS - Usatei](https://osu.ppy.sh/beatmapsets/275151#osu) by [Damnjelly](https://osu.ppy.sh/users/1666355)  
+190 BPM, 3:55 | 7.99★  
+*written by [waywern2012](https://osu.ppy.sh/users/5870453)*
+
+Usatei is coming back with a fairly aged map from Damnjelly. In 2015, the map had some minor changes done to it, but it was left alone after some time. The map plays really well even by today's standards and has interesting jumps, note placements and spaced streams. With a fair bit of mix with spaced linear triples the map has, it is fairly captivating for higher skilled players. Are you ready to jump around in this amazing map?
```

Comment:
"With a fair bit of mix with spaced linear triples the map has"

I'm not sure what this is saying.